scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -49,7 +53,7 @@ fun main(vararg args: String) {
   ScriptBackgroundCoroutineDispatcher().use { scriptBgDispatcher ->
     val processTimeout: Long = args.find { it.startsWith("processTimeout=") }
       ?.substringAfter("=")
-      ?.toLongOrNull() ?: 5
+      ?.toLongOrNull() ?: 10
```

Comment:
Before reverting the change, I wanted to explain my reasoning behind the `processTimeout` update in **RunCoverage.kt**.

During local coverage analysis, I often encountered files in large packages taking over **300 seconds** to process, leading to **"process didn't complete within the intended time"** errors. 

![image](https://github.com/oppia/oppia-android/assets/122200035/f19afe5f-4fe8-492b-ac3d-be7e3b2c1a8d)

To address this, I proposed increasing the timeout to 10 minutes _(by default to give ample time)_. This duration can vary depending on the files and the device and having the `processTimeout` configurable via command-line arguments should be useful for this.

Would you prefer keeping it at 5 minutes and allowing developers to adjust the processTimeout through command-line arguments if needed, or can we set it to 10 minutes as a middle ground? Would like to confirm how it could be done, and I will revert the changes as guided.

Thanks.

Reply:
Frankly, I only mention 5 minutes becase it is consistent with the rest of the test configuration, and since we allow the developer to configure a timeout. Configuring a timeout is agreat workaround and the duration wil depend on a developer's machine performance.

Reply:
Got it! I've reverted it to **5 minutes** for consistent timing _(considering we do have the configuration available)_. Thanks!